### PR TITLE
Header design

### PR DESCRIPTION
**Pull request by joshuaogle**
Mobile styles are still really basic, but it's a start.

screen shot 2015-06-26 at 1 30 53 pm

### CODE REVIEW

File: source/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  .container {
+    @include media($large-screen) {
+      display: flex;
+    }
+  }
+
+  .header-nav {
+    @include media($large-screen) {
+      display: flex;
+      align-items: center;
```
Comment (donokuda) [inline source/stylesheets/components/_header.scss:15]: Should this be above `display: flex`?

File: source/stylesheets/components/_header.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  .st-search-input {
+    appearance: none;
+    background: image-url("search-icon.svg") rgba(white, 0.5) 10px 10px no-repeat;
+    border: none;
+    border-radius: 35px;
+    box-shadow: none;
+    color: $white;
+    line-height: 35px;
+    height: 35px;
+    font-size: 0.9rem;
```
Comment (donokuda) [inline source/stylesheets/components/_header.scss:65]: :abc:?

**Comment by donokuda** [comment]
This LGTM :+1:

**Comment by joshuaogle** [comment]
Fixed those. Thanks, Don!

**Pull request merged**